aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -420,13 +596,23 @@ def export_tree(
 
         progress_bar.update()
 
-    # We will iteratively explore the AiiDA graph to find further nodes that should also be exported.
-    # At the same time, we will create the links_uuid list of dicts to be exported
+    return entities_starting_set, given_node_entry_ids
+
+
+def collect_export_nodes(given_node_entry_ids: Set[int], silent: bool,
+                         **traversal_rules: bool) -> Tuple[Set[int], Dict[int, str], List[dict], Dict[str, bool]]:
+    """Iteratively explore the AiiDA graph to find further nodes that should also be exported
+
+    At the same time, we will create the links_uuid list of dicts to be exported
+
+    :param silent: suppress console prints and progress bar.
+    """
+    from aiida.tools.graph.graph_traversers import get_nodes_export
```

Comment:
I guess this should go to the top of the file? wondering why pylint does not complain...

Reply:
note already fixed in #4448

Reply:
Ok!